packages/db/src/database.ts

```diff
@@ -12,15 +12,15 @@ export interface DatabaseConfig {
 export class Database extends Dexie {
   accounts!: Table<Account>
   clusters!: Table<Cluster>
-  preferences!: Table<Preference>
+  settings!: Table<Setting>
```

Comment:
@tobeycodes I'm inclined to make this singular. `db.setting.add()` vs `db.settings.add()`. Thoughts?

Reply:
I have more commonly used plurals for database tables

Reply:
Ok, leaving it as is.